### ISSUE TITLE
Cant get working youtube live

### ISSUE THREAD

**Issue by saritikon** [datarhei/restreamer#816]
No youtube live video

Using Restreamer on Virtualbox Ubuntu server (ubuntu-24.04.1-live-server-amd64), restreamer-v2.12.0 (datarhei-core v16.16.0 (linux/amd64) - gentle-limit-3866)
Network card in Virtualbox configured as Bridge, so Ubuntu gets local ip (192.168.5.132) from router the same way like my desktop pc

Source:
Using Hikvision IP camera DS-2DE7A825IW-AEB(T5) (192.168.5.64)
Camera config: 
Mainstream
Videotype: Video Stream
Resolution: 3840*2160
Bitrate VBR
Video Quality: Higest
Video Frame Rate(fps): 1
Max. Bit Rate 16384
Average Bit Rate 8192
Video Encoding H.264
H.264+ Off
Profile High
I-Frame Interval 50
SVC Off
Smoothing 50

Hikvision camera configured in Restreamer via RTSP, using passthrough method, no audio
Videostream from hikvision camera in Reastreamer ui, plays fine, CPU load 0-2%

Destination:
YouTube Live
Delivering mode HLS

Source & Encoding
Source HLS (msmfs)

Select your encoding setting:
passthrough (copy)

No videostream in youtube studio

**Comment by ioppermann** [comment]
For your resolution (4K / 2160p) of your camera you should provide a "Video Frame Rate" of 30 (currently you have 1) and an "I-Frame Interval" of 60. The recommended bitrate for that resolution is 30000 kbit/s (see https://support.google.com/youtube/answer/2853702).

Also you should add an audio stream and try the RTMP delivery method. The HLS delivery is still experimental.

**Comment by saritikon** [comment]
Hi @ioppermann 
Thanks for your quick response!

Camera doesn't support 30 or 60 fps, only 25 fps max

Added an audio in Restreamer with white noise and it works both with HLS and RTMP! 
So, i got the working setup:
Settings in hikvision admin panel: videostream only (no audio) resolution 4k, 10fps, higest video quality, VBR with 8192, encoding H264, I-frame 20, H264+ is off, SVC off

Also ok with uploaded mp3 looped sound
So the problem was in no sound setup.

Now trying to reduce traffic, since i have an almost static picture (nature)

Thank you!

**Comment by ioppermann** [comment]
If you want to reduce traffic, then only reducing the bit rate will help. You can even set the FPS to 25.

**Comment by saritikon** [comment]
Yes, i changed setting back to 25 fps and traffic remains same. Thank you!
And with bitrate 2048 i got near 1,850 kbit/s

By the way, i noticed that here https://support.google.com/youtube/answer/2853702 a H265 codec is seem to supported and widely used by youtube? So any chance for support H265 in Restreamer?

**Comment by ioppermann** [comment]
We're working on passthrough support for other codecs as well.
